lib/twingly/search/error.rb

```diff
@@ -16,7 +18,10 @@ def self.from_api_response(code, message)
       end
     end
 
-    class AuthError < Error
+    class AuthenticationError < Error
```

Comment:
Just an idea: if we don't want any breaking changes we could keep `AuthError` around and have both `AuthenticationError` and `AuthorizationError` inherit it.

Reply:
Sure! That's easier to roll out, then maybe we could just leave `AuthError` in LiveFeed as is?

Reply:
Yes, sure. We can always add an `AuthenticationError` subclass to LiveFeed later if that is something we decide we want.

Reply:
@roback like 43c6388?

Reply:
If we think that's enough, I might as well add `AuthenticationError < AuthError` to LiveFeed as well.

Reply:
> @roback like [43c6388](https://github.com/twingly/twingly-search-api-ruby/commit/43c63887c90f8dc8b6e356df2fef56e2700c977d)?

:+1: Yes, that's exactly what I was thinking.

We can add it to LiveFeed now as well as that makes things less confusing for us/the users in the future.

Reply:
Yeah